cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/chart_test.go

```diff
@@ -463,24 +463,7 @@ func TestGetAvailablePackageVersions(t *testing.T) {
 				AvailablePackageRef: availableRef("bitnami/redis", "kubeapps"),
 			},
 			expectedStatusCode: codes.OK,
-			expectedResponse: &corev1.GetAvailablePackageVersionsResponse{
-				PackageAppVersions: []*corev1.PackageAppVersion{
-					{PkgVersion: "14.6.1", AppVersion: "6.2.4"},
-					{PkgVersion: "14.6.0", AppVersion: "6.2.4"},
-					{PkgVersion: "14.5.0", AppVersion: "6.2.4"},
-					{PkgVersion: "14.4.0", AppVersion: "6.2.4"},
-					{PkgVersion: "13.0.1", AppVersion: "6.2.1"},
-					{PkgVersion: "13.0.0", AppVersion: "6.2.1"},
-					{PkgVersion: "12.10.1", AppVersion: "6.0.12"},
-					{PkgVersion: "12.10.0", AppVersion: "6.0.12"},
-					{PkgVersion: "12.9.2", AppVersion: "6.0.12"},
-					{PkgVersion: "12.9.1", AppVersion: "6.0.12"},
-					{PkgVersion: "12.9.0", AppVersion: "6.0.12"},
-					{PkgVersion: "12.8.3", AppVersion: "6.0.12"},
-					{PkgVersion: "12.8.2", AppVersion: "6.0.12"},
-					{PkgVersion: "12.8.1", AppVersion: "6.0.12"},
-				},
-			},
+			expectedResponse:   expected_versions_redis,
```

Comment:
Up to you, but test cases are the one place where I personally think it's worth being verbose, so when you come back, you don't need to scroll through the file to find out what the expectation is, but it's clearly readable as part of each test case.

Reply:
well, then let's agree to disagree on this one. I do see your point. However, while I am actively developing I find that when a test case has a dozen or so scenarios and I only need to debug/look in depth at one or two of them, and need to comment the other ones out. It's a lot easier to do so when the cases are small code line-wise, and a painful to do when there is a lot of scrolling to do and hard to get the open/close comments into the right place. So this style is a trade-off I chose: there are pluses and minutes, and it's just a matter of more pluses than minuses

Reply:
Yep, makes sense :)